packages/uniforms-antd/src/ListField.tsx

```diff
@@ -48,43 +49,57 @@ function List({
   wrapperCol,
   ...props
 }: ListFieldProps) {
+  const labelNode = !!label && (
+    <div>
+      {label}
+      {!!info && (
+        <span>
+          &nbsp;
+          <Tooltip title={info}>
+            <QuestionCircleOutlined />
+          </Tooltip>
+        </span>
+      )}
+    </div>
+  );
+
+  const borderStyle = {
+    ...(error ? { borderColor: 'rgb(255, 85, 0)' } : {}),
+    ...style,
+  };
```

Comment:
The presence of both `error` and `style` is rather rare, maybe we could optimize for this case? Something like this:
```suggestion
  const borderStyle = 
    error
      ? { borderColor: 'rgb(255, 85, 0)', ...style }
      : style;
```

This would create a new object only when it's needed.